pkg/channel/consolidated/dispatcher/dispatcher.go

```diff
@@ -141,7 +141,11 @@ func (d *KafkaDispatcher) ServeHTTP(w nethttp.ResponseWriter, r *nethttp.Request
 }
 
 func NewDispatcher(ctx context.Context, args *KafkaDispatcherArgs) (*KafkaDispatcher, error) {
+	confTemplate := sarama.NewConfig()
+	confTemplate.Consumer.Offsets.Initial = sarama.OffsetOldest
```

Comment:
what happens on upgrades (e.g. from 0.21 to 0.22), when this changes?

Does it have side effects.

I shot too quick w/ the LGTM

Reply:
https://github.com/knative-sandbox/eventing-kafka/pull/428#issuecomment-789802302